python/offline_reconstruction.py

```diff
@@ -5,15 +5,14 @@
 import urllib
 import warnings
 
-import torch
 import tqdm
 
 from data.provider import DataProvider
 from data.rectimestamps import TimestampProviderFile, TimestampProviderRate
-from reconstruction.utils.loading_utils import load_model, get_device
-from reconstruction.image_reconstructor import ImageReconstructor
-from reconstruction.options.inference_options import set_inference_options
-from reconstruction.utils.voxelgrid import VoxelGrid
+
+import e2vid
+from e2vid.options.inference_options import set_inference_options
+from e2vid.utils.voxelgrid import VoxelGrid
 
 
 def download_checkpoint(path_to_model):
```

Comment:
function duplicate (already defined and used here: python/reconstruction/e2vid/__init__.py)

Reply:
fixed in latest commit